microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,255 @@
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
+                    }
+                },
+                {
+                    "platform": "android",
+                    "attestations": {}
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+<!-- ---------- -->
+###### Important fields and values
+
+* `"platform":` must be `"edge"` or `"android"`.
+
+* `"attestations":` must include `"attribution_reporting_api"`, `"shared_storage_api"`, `"private_aggregation_api"`, and `"ad_selection_api"`.
+
+   Each `"attestations":` entry must have a single field, `"ServiceNotUsedForIdentifyingUserAcrossSites":`, with either a `true` or `false` value.  `true` means that this service is not used for identifying the user across sites.  `false` means that this service is used for identifying the user across sites.
+
+* `"ownership_token":` is the OT token generated for your individual domain registration.
+
+
+<!-- ====================================================================== -->
+## Deploy the Ad Selection API services
+
+The Ad Selection API uses a trusted execution environment (TEE) to provide a level of assurance for data integrity, data confidentiality, and code integrity; see [Confidential Computing: Hardware-Based Trusted Execution for Applications and Data](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/confidentialcomputing_outreach_whitepaper-8-5x11-1.pdf).  Services provided by the Ad Selection API must run in a TEE to secure the data used by these services.
+
+Ad Selection Services running in a TEE should be deployed on a cloud platform that supports the necessary security features.  Initially, services can be deployed in Azure using [Confidential ACI containers](/azure/container-instances/container-instances-confidential-overview).
+
+The Ad Selection API provides different services that need to be deployed by sellers or buyers.
+
+
+<!-- ------------------------------ -->
+#### Images for each service
```

Comment:
review new "Images" section.
* first sentence is from above paragraph, in draft Doc.
* links are from See Also h2 section at end, in draft Doc.
* statement about "must" is from See Also section at end of draft Doc (such detail is out of place in See Also and belongs in article body):
![image](https://github.com/user-attachments/assets/f096c238-42e4-4c3a-8c6b-b7441b6c5ade)

Reply:
i think it's ok to close comment, b/c PR is out for review, & this change is normal Writer/Editor formatting that just gathers info that wasn't properly placed; not new content or statements.